instrumentation/http-url-connection/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/httpurlconnection/HttpMethodAttributeExtractor.java

```diff
@@ -40,7 +42,11 @@ public void onEnd(
     if (getOutputStreamContext.isOutputStreamMethodOfSunConnectionCalled()) {
       String requestMethod = connection.getRequestMethod();
       // The getOutputStream() has transformed "GET" into "POST"
-      attributes.put(SemanticAttributes.HTTP_METHOD, requestMethod);
+      attributes.put(
+          SemconvStability.emitStableHttpSemconv()
+              ? HttpAttributes.HTTP_REQUEST_METHOD
```

Comment:
`HttpCommonAttributesExtractor` checks agains known methods before setting `HTTP_REQUEST_METHOD`. Could do this here also, currently skipped because `HttpURLConnection` only accepts methods that are in our known list.

Reply:
the list of known methods could potentially be reduced via `otel.instrumentation.http.known-methods`